install.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -e
+
+echo "Installing Screenpipe..."
+
+BASE_URL="https://api.github.com/repos/mediar-ai/screenpipe/releases"
+DESTINATION=${SCREENPIPE_PATH:-"$HOME/screenpipe"}
+TEMP_DIR=$(mktemp -d)
+
+# Detect operating system and architecture
+OS=$(uname -s)
+ARCH=$(uname -m)
+case $OS in
+"Darwin")
+  if [[ "$ARCH" == "arm64" ]]; then
+    PATTERN="aarch64-apple-darwin"
+  else
+    PATTERN="x86_64-apple-darwin"
+  fi
+  ;;
+"Linux")
+  PATTERN="x86_64-unknown-linux-gnu"
+  ;;
+*)
+  echo "Error: Unsupported operating system $OS"
+  exit 1
+  ;;
+esac
+
+# Fetch the latest release dynamically
+echo "Fetching the latest release information from GitHub..."
+RELEASE_INFO=$(curl -fsSL "$BASE_URL")
+LATEST_RELEASE=$(echo "$RELEASE_INFO" | jq -r 'map(select(.prerelease == false)) | first | .tag_name')
```

Comment:
this will fail if user hasn't installed jq & not all unix user uses jq!
maybe you'd use some gnu core stuff!

Reply:
> @tribhuwan-kumar Can you help in testing this. Installation part looks good but need to check with `screenpipe` command once done. I can't install the openssl3 with `sudo apt install libssl3` just need to check with it
> 
> ```
> ...........
> Screenpipe installed successfully!
> Run 'screenpipe --help' to get started.
> Installation completed.
> @Yashashwini2003 ➜ /workspaces/screenpipe (feature/install) $ screenpipe
> screenpipe: error while loading shared libraries: libcrypto.so.3: cannot open shared object file: No such file or directory
> @Yashashwini2003 ➜ /workspaces/screenpipe (feature/install) $ 
> ```

not getting that error

https://github.com/user-attachments/assets/c8b96223-61c9-4127-8dcf-d7c8a7e29835